examples/alloy-editor/app.js

```diff
@@ -1,11 +1,14 @@
-import widget from '../../lib/api'
+import { init } from '../../lib/api'
+
+const AlloyEditor = window.AlloyEditor
```

Comment:
I would rather keep `const` since I want the program to throw if someone reassigns `AlloyEditor`.